config/routes.rb

```diff
@@ -1,11 +1,24 @@
 Rails.application.routes.draw do
-  resources :events
-  resources :players
-  resources :calendar, only: [:index,:show]
+  resources :events do
+    member do
+      get 'edit_player'
```

Comment:
maybe we can use `resources` here also, just restrict to only edit and update actions and point to a proper controller